exampleSite/content/_index/items-2/extendible.md

```diff
@@ -0,0 +1,7 @@
++++
+title = "Extendible"
+weight = 10
+icon = "fas fa-boxes"
++++
+
+With Hugo's resource fallthrough you can create and update fragments and Syna will use the new or updated version of those fragments
```

Comment:
Seems like a small thing, more in line with general flexibility.

Reply:
I was thinking that flexibility in Syna is that you can reorder nav links, change background colors and other parameters for every fragment. But with extensibility I mean that Syna will use any fragment and it's own fragments can be updated which is actually a Hugo feature but for the uninitiated, it's just another attraction.